rskj-core/src/main/java/co/rsk/peg/PegUtils.java

```diff
@@ -35,4 +36,8 @@ public static PegTxType getTransactionType(
             return PegTxType.UNKNOWN;
         }
     }
+
+    public static boolean isAnyUTXOAmountBelowMinimum(Coin minimumPegInTxValue, BtcTransaction btcTx, Wallet wallet) {
```

Comment:
Any reason to have this here instead of sending directly to the PegUtilsLegacy?

Reply:
Yes, the bug in the current `isAnyUTXOAmountBelowMinimum`  method, which returns true when passing a btcTx with no utxo. The idea is to refactor this method in another ticket(maybe the name of the method will be changed), so since the RSKIP379. no utxos or any utxo amount below the minimum will return false.